### PR TITLE
fixes #1380 Refactor social links styling

### LINKED ISSUE #1380: Non-Visibility of the X logo when switched to dark mode
**Describe the bug**

When switched to dark mode the logo of the X is not visible

**To Reproduce**
1. Go to Footer
2. Click on Dark Mode switch
3. See error

**Expected behavior**

There should be a background of white to show the logo perfectly

**Screenshots**
{6D08D9B8-7715-4DDC-AC41-0A34C7B560CC}

{0EE6E55F-8014-4FF9-92AC-286195A7C200}

**Additional context**

### PR DESCRIPTION
**Pull request by harjasae2001**
### Pull Request Checklist

- [X] I have added screenshots and videos to show before and after the working of my code.
- [X] I have ensured that the screen size is set to 100% while making the video.
- [X] I have synced the latest fork with my local repository and resolved any conflicts.
- [X] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [X] I understand that if any the above conditions are not met , my PR will not be MERGED .

https://screenrec.com/share/esxnWDy87d

### CODE REVIEW

**Comment by ANSHIKA-26** [comment]
issue number ?

File: blog.html [hunk 1]
```diff
@@ -539,7 +538,7 @@ <h3 class="form-title">Share Your Thoughts</h3>
           <label for="commentText" class="input-label">Your Comment</label>
         </div>
         <input type="hidden" id="parentCommentId" value="">
-        <button type="submit" class="submit-button" onclick="saveComment(event)">
```
Comment (ANSHIKA-26) [inline blog.html:542]: why did you remove the onClick Js from here ?

File: blog.html [hunk 2]
```diff
@@ -1279,11 +1278,11 @@ <h2 class="footer-title secondary-title">Popular Tags</h2>
 
           <div class="tags">
             <div class="d-flex-gap flex-wrap">
-              <a href="#" class="nav-link btn bg-light pop-link" >Travel</a>
-                           <a href="#" class="nav-link btn bg-light pop-link" >Food</a>
-                           <a href="#" class="nav-link btn bg-light pop-link" >Lifestyle</a>
-                           <a href="#" class="nav-link btn bg-light pop-link" >Techonogy</a>
-                           <a href="#" class="nav-link btn bg-light pop-link" >Fashion</a>
+              <a href="#" class="nav-link btn bg-light pop-link" style="font-weight: 400;">Travel</a>
```
Comment (ANSHIKA-26) [inline blog.html:1281]: why ?

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by ANSHIKA-26** [comment]
@harjasae2001 on going through your code , I understand that you have not synced your fork and made a PR , I will be closing this PR , kindly refork , resync the repo and then make the changes , also , you have already ticked the checkbox which says that your fork is synced , make sure to follow the instructions rather than just ticking the checkbox

File: blog.html (hunk 1 above)
Comment (harjasae2001) [reply blog.html:542]: Maybe there is some fork problem from my end. I will refork it.

**Comment by harjasae2001** [comment]
Sure, i will update my fork and then create a new PR from my end. Sorry for inconvienence.
